code/Style.md

```diff
@@ -70,6 +72,28 @@ func Example() (err error) {
 }
 ```
 
+Sometimes, there are situations where a function, calls another function that only needs to return the value or the error returned by the call; when the called function return errors not created or wrapped by `errs` package the error must be  wrapped or annotated, however it isn't needed to have a conditional to achieve such purpose, because the functions that `errs` package have ignore the `nil` value when they are passed through; for example
```

Comment:
Let's list exceptions here where we shouldn't use wrapping.

1. Read/Write etc. expect `io.EOF` returned.
2. some packages require specific error codes

Reply:
I'm aware of those now, this PR was too old and I didn't reread, so good point, I will add them